src/ralph/back_office/tests/test_models.py

```diff
@@ -271,6 +275,26 @@ def setUp(self):
         messages = FallbackStorage(self.request)
         setattr(self.request, '_messages', messages)
 
+    def test_move_as_data_center_asset(self):
+        bo_asset = BackOfficeAssetFactory()
```

Comment:
please add unit test where there is some transition history and attachments attached to asset (we've to make sure that this will be visible from new object too).